README.md

```diff
@@ -86,6 +86,15 @@ edgelabel = 1:Graphs.ne(g)
 gplot(g, edgelabel=edgelabel, nodelabel=nodelabel, edgelabeldistx=0.5, edgelabeldisty=0.5)
 ```
 
+## Add dash style to edges
+```julia
+using Measures
+gplot(g, edgedashstyle=[5mm, 2mm])
```

Comment:
The use of this kwarg should be properly documented. In this case, a dash with size 5mm is created with 2mm gaps.

Reply:
No, all edges will have the same style, that is alternating 5mm solid line and 2mm blank line.